hash/bbc_flop_torch.xml

```diff
@@ -3,7 +3,7 @@
 
 <!-- Acorn BBC Micro - Torch Z80/68000 Co-Processor discs -->
 
-<softwarelist name="bbc_flop_torch" description="Acorn Torch Z80/68000 Co-Processor discs">
+<softwarelist name="bbc_flop_torch" description="Acorn (Torch) Z80/68000 Co-Processor discs">
```

Comment:
These are intended for use with Torch computers, though can also be used with BBC Micro with Torch board fitted. Adding Acorn for the sake of ordering and demoting Torch to brackets is not needed.
Please leave my softlists alone.